.node-version

```diff
@@ -1 +1 @@
-20.6.0
```

Comment:
node 22 ?a

Reply:
I kept everything as it is on the original repo, to keep the changes in step with them as you wanted